subprojects/dependency-management/src/test/groovy/org/gradle/internal/locking/LockFileReaderWriterTest.groovy

```diff
@@ -142,6 +142,26 @@ empty=d
         1 * listener.fileObserved(lockFile)
     }
 
+    def 'reads an invalid unique lock file '() {
```

Comment:
❌ We need a test similar to this one but for a different scope of the file. See "reads a unique lock file with prefix" for inspiration.